pycaret/internal/plots/time_series.py

```diff
@@ -450,14 +450,36 @@ def plot_acf(
     data_kwargs: Optional[Dict] = None,
     fig_kwargs: Optional[Dict] = None,
 ) -> PlotReturnType:
-    """Plots the ACF on the data provided"""
-    fig, return_data_dict = None, None
-
+    """Plots the ACF on the data provided
+
+    Parameters
+    ----------
+    data : pd.Series
+        Data whose correlation plot needs to be plotted
+    fig_defaults : Dict[str, Any]
+        The defaults dictionary containing keys for "width" and "height" (mandatory)
```

Comment:
Thanks for adding the docstring in the right numpy format. I have been meaning to do that for a while now.

I don't think these are mandatory. Plus it may not necessarily be limited to "width" and "height". You could also have template, etc. I would just say, something like "The defaults settings for the plotly figure."